test/end-to-end-testapp/settings.js

```diff
@@ -1,5 +1,5 @@
 var SDKsettings = {
-    apiKey: 'testAPIKey'
+    environmentId: 'fyi3PTZxk'
```

Comment:
if you are okay with including this code in your repo, then it's fine to keep this, but if this is a sensitive key,  you should use an environment variable to keep things like this a secret.  but i know API keys are public so this may not matter for you